src/anbox/cmds/session_manager.cpp

```diff
@@ -157,7 +160,8 @@ anbox::cmds::SessionManager::SessionManager()
     auto platform = platform::create(utils::get_env_value("ANBOX_PLATFORM", "sdl"),
                                      input_manager,
                                      display_frame,
-                                     single_window_);
+                                     single_window_,
+                                     no_touch_emulation_);
```

Comment:
We should create a platform::Configuration struct and pass that instead here to avoid having too many arguments. The Configuration struct can take display_frame, single_window_ and no_touch_emulation_

Reply:
ok, having too many arguments was one of my concern